relay-server/src/actors/envelopes.rs

```diff
@@ -416,6 +416,28 @@ fn with_tag(
     tags
 }
 
+#[cfg(feature = "processing")]
+fn get_measurement_rating(name: &str, value: f64) -> String {
+    let rate_range = |meh_ceiling: f64, poor_ceiling: f64| {
+        debug_assert!(meh_ceiling < poor_ceiling);
+        if value < meh_ceiling {
+            "good".to_owned()
+        } else if value < poor_ceiling {
+            "meh".to_owned()
+        } else {
+            "poor".to_owned()
+        }
+    };
+
+    match name {
+        "measurement.lcp" => rate_range(2500.0, 4000.0),
+        "measurement.fcp" => rate_range(1000.0, 3000.0),
+        "measurement.fid" => rate_range(100.0, 300.0),
+        "measurement.cls" => rate_range(0.1, 0.25),
+        _ => "unknown".to_owned(),
```

Comment:
Should we return `None` here instead, and only insert the tag if `Some`?

Reply:
i prefer this because then you know for which buckets this classification ran at all